python-frontend/src/main/java/org/sonar/python/SubscriptionVisitor.java

```diff
@@ -140,6 +155,16 @@ public PythonFile pythonFile() {
       return pythonVisitorContext.pythonFile();
     }
 
+    @Override
+    public Map<String, Set<Symbol>> getTypeShedSymbols() {
```

Comment:
Because they don't reference only typeshed symbols, maybe I'd rename it to `stubFilesSymbols()`